cmd/edit/kubeletconfig/cmd.go

```diff
@@ -124,6 +120,12 @@ func run(_ *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if args.podPidsLimit > maxPidsLimit {
+		r.Reporter.Errorf("The maximum value for --pod-pids-limit is '%d'. You have supplied '%d'",
+			maxPidsLimit, args.podPidsLimit)
+		os.Exit(1)
+	}
```

Comment:
Will this error/return ever be reached? The interactive prompt you made in [this file](https://github.com/openshift/rosa/blob/b1410776cbd63618511efbab573dd7bdd079264a/pkg/kubeletconfig/config.go) has validations for this same scenario

Reply:
@hunterkepley Yes, it's reached if I don't use interactive mode and simply pass the `--pod-pids-limit` flag e.g `rosa create/edit kubeletconfig -c rblake-test --pod-pids-limit=5000`